integration-tests/features/steps/stack_analysis.py

```diff
@@ -650,6 +650,19 @@ def check_analyzed_dependencies_count(context, expected=1):
             analyzed_dependencies_count, expected)
 
 
+@then("I should find exactly one unknown dependency")
```

Comment:
You can never be sure an unknown will remain unknown for a lifetime.

Reply:
We already have the following test step:
`Then I should find no more than X unknown dependencies`

The one with exact one unknown dependency is to check the following file https://github.com/fabric8-analytics/fabric8-analytics-common/pull/988/files#diff-c1053ef11641602c7a94f1ba11fc57c3 ie the one where component is non existent (we can use even more unlike name of course :)

Reply:
Ok, in that case we could rename this to something like
`@then("I should find exactly one really unknown dependency")`

Reply:
Ok I'm updating the PR. Thanks @miteshvp